### ISSUE TITLE
arduino 1.8.8 and atmeg328p 

### ISSUE THREAD

**Issue by KKuKK** [carlosefr/atmega#10]
Hi 
I want to use your code for a stand alone atmeg328p at 8 Mhz intern with Arduino 1.8.8
I included your json files.
A can use my code on an Arduino Version 1.0.5 without problem, I can also use a Arduino Mini oder Nano. All works, but 
the code is round about 2kb bigger, or the available flash is only 30700 instead of 32768 when I use this 
settings here.

The problem is, all works fine, compiling and uploading, but the chip is dead. It ist just not running.
Not even a simple blink program. 
What can go wrong by usings your code? 

There are just 2 options
Chip : atmega328p
Clock: 8mhz intern.
Fuses are ok. It works if I use other boards.

I realy would like to use your code, because it gives me 2 kb more flash.
Would be great if you have any idea.
Thank you.
Greatings  klaus

**Comment by carlosefr** [comment]
The chip works if you use it in a board, is that what you mean by the fuses? Or you mean you used "burn bootloader" to set the fuses according to your selection before the first upload, and it still doesn't work?

**Comment by KKuKK** [comment]
Hi
Great to hear from you so fast! Thanks.
The second is write. I use extremburner or ponyprog and an usbasp programmer.
No bootload, for space. The chip is standing alone, 8 mhz intern.
All works, if i use arduino 1.05 and the old style for board.txt file.
If I use arduino 1.8.8 and your board setting, then the chip is dead.
If I use arduino 1.8.8 with other boards as nano, or mini, all works, but
I loose 2kb flash.
I tried 4 different chips, which all work in there circuit, but all are dead, if
I use your board settins:
Atmega328p
8mhz intern
There are no more options.
Quit crazy, hope we find out.
Thanks a lot for help
Greetings klaus 

Von meinem iPad gesendet

**Comment by KKuKK** [comment]
Hi again!

I think I am becoming crazy, I just tried it no again, for at least 20. time.

All of a suddon, it works !!!!

I have no idea, I changed nothing. Same settings. 

So sorry, realy have no explanation.

I have tried it so often, finaly a gave up, now it works.

So what, I am happy.

Thanks you for your answer!
Greetings klaus

**Comment by KKuKK** [comment]
Sorry for complaining, but it was real, thanks for your help.
greetings

**Comment by carlosefr** [comment]
The most common reason is changing the settings in the menu (processor, clock) but forgetting to "burn bootloader". Unfortunately, the Arduino IDE does not check if the fuses match the menu selection before uploading.

**Comment by KKuKK** [comment]
Thanks for your answer!
I realy cannot give you an explanation. I tried it several times month ago and gave up.
It was strange. I did not change anything at the FUSE and the board settings were 
always the same correct setting.
When I used the Nano board with 8 Mhz intern it works.
When I use the same setting with your boards seconds later, it didn’t work.
Until today. 
Hope I will find the reason, my be per accident. 
Right now I am quite happy, my remaining flash is up from 3 to 5 kb
quit a lot.
Thanks again.
I will mail you if I find the reason.

An auto check form settings and fuse would be realy helpful.
But in this case, I am sure it was not the reason for the problem.

greetings Klaus
